vector/src/main/java/im/vector/app/features/location/LocationSharingFragment.kt

```diff
@@ -168,6 +171,16 @@ class LocationSharingFragment @Inject constructor(
                 .show()
     }
 
+    private fun handleLiveLocationSharingNotEnoughPermission() {
+        MaterialAlertDialogBuilder(requireActivity())
+                .setTitle(R.string.live_location_not_enough_permission_dialog_title)
+                .setMessage(R.string.live_location_not_enough_permission_dialog_description)
+                .setPositiveButton(R.string.ok) { dialogInterface, _ ->
+                    dialogInterface.dismiss()
```

Comment:
I do not think this code is necessary. You can replace by 
```kotlin
.setPositiveButton(R.string.ok, null)
```

Reply:
Of course, done.